docs/3.0rc/get-started/quickstart.mdx

```diff
@@ -93,7 +93,7 @@ def get_contributors(repo_info: dict):
     return contributors
 
 @flow(log_prints=True)
-def repo_info(repo_owner: str = "PrefectHQ", repo_name: str = "prefect"):
+def log_repo_info(repo_owner: str = "PrefectHQ", repo_name: str = "prefect"):
```

Comment:
this flow declares a variable also named `repo_info`, which is confusing and bad practice